src/kube-runtime/src/init

```diff
@@ -78,6 +86,17 @@ PAI_RUNTIME_DIR=${PAI_WORK_DIR}/runtime.d
 
 PAI_LOG_DIR=${PAI_WORK_DIR}/logs/${FC_POD_UID}
 
+# Move previous logs to another folder. Notice: for init.log, some part will append to previous log file
+LOG_FILES=$(find $PAI_LOG_DIR -maxdepth 1 -type f)
+if [[ ! -z $LOG_FILES ]]; then
+  PRE_LOG_DIR=$PAI_LOG_DIR/prelog_$(date +%s)
```

Comment:
>$(date +%s) [](start = 34, length = 11)

time in second may be the same. So why not just use the LOG_FILES count as suffix?